repos/spack_repo/builtin/packages/py_bottleneck/package.py

```diff
@@ -29,4 +31,14 @@ class PyBottleneck(PythonPackage):
     depends_on("py-versioneer", when="@1.3.3:", type="build")
     depends_on("py-numpy", type=("build", "run"))
     # https://github.com/pydata/bottleneck/issues/453
-    depends_on("py-numpy@:1", when="@:1.3", type=("build", "run"))
+    depends_on("py-numpy@2:", type=("build", "run"), when="@1.5:")
```

Comment:
Still don't see a lower bound on numpy 2. It's present in the build-time deps, but not in the run-time deps. The build-time dep lower bound is only needed to build a binary that is compatible with both. At run-time, either should work.